pom.xml

```diff
@@ -44,10 +44,10 @@
     <url>http://www.forgerock.org</url>
 
     <properties>
-        <ob-common.version>1.2.10</ob-common.version>
-        <ob-clients.version>1.2.10</ob-clients.version>
-        <ob-jwkms.version>1.2.9</ob-jwkms.version>
-        <ob-auth.version>1.1.9</ob-auth.version>
+        <ob-common.version>1.2.11</ob-common.version>
+        <ob-clients.version>1.2.11</ob-clients.version>
+        <ob-jwkms.version>1.2.10</ob-jwkms.version>
+        <ob-auth.version>1.1.10</ob-auth.version>
```

Comment:
Auth needs releasing as well no? It uses commons... which needs to be at 1.2.11 in there or we risk version clashes??

Reply:
The Auth has been release one hour ago:
- https://github.com/OpenBankingToolkit/openbanking-auth/releases/tag/1.1.10